### PR TITLE
[AUC-8] Header

### PR DESCRIPTION
**Pull request by Trevoule**
header revised

### CODE REVIEW

File: components/Header/components/HeaderMenu.js [hunk 1]
```diff
@@ -1,37 +1,45 @@
...
+  return (
+    <StyledMenu anchorEl={anchorEl} open={!!anchorEl} onClose={handleClose}>
+      {MenuSettings.map(({ title, StartIcon, color = "text.primary" }) => (
+        <MenuItem
+          onClick={handleClose}
+          disableRipple
+          key={title}
+          sx={{ backgroundColor: "background.footer" }}
+        >
+          {/* <Link href={href}> */}
```
Comment (sergeevds) [inline components/Header/components/HeaderMenu.js:33]: ?

File: react/src/components/Header/index.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            }}
+          >
+            <IconButton
+              edge="start"
+              color="inherit"
+              aria-label="menu"
+              sx={{ mr: 4 }}
+            >
+              <MenuIcon />
+            </IconButton>
```
Comment (sergeevds) [inline react/src/components/Header/index.js:55]: not by design

File: react/src/components/Theme/components/MuiInput.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// import { palette } from "../palette";
+
+export const MuiInput = {
+  styleOverrides: {
+    root: {
+      borderRadius: "14px !important",
+      // width: 400,
```
Comment (sergeevds) [inline react/src/components/Theme/components/MuiInput.js:7]: <Box width={400}>
    <TextField fullWidth/>
</Box>

Comment (Trevoule) [reply react/src/components/Theme/components/MuiInput.js]: fixed

File: components/Header/components/HeaderMenu.js (hunk 1 above)
Comment (Trevoule) [reply components/Header/components/HeaderMenu.js]: fixed

**Pull request merged**